Diabetes/TabBarControllers/Dashboard/APHDashboardViewController.m

```diff
@@ -281,45 +277,45 @@ - (void)preparingScoringObjects
     [self.insightAndScoringQueue addOperationWithBlock:^{
         HKQuantityType *hkQuantity = [HKQuantityType quantityTypeForIdentifier:HKQuantityTypeIdentifierStepCount];
         weakSelf.stepScoring = [[APCScoring alloc] initWithHealthKitQuantityType:hkQuantity
```

Comment:
In block, convert to strongSelf to retain the object, then use strongSelf.
This prevents object being deallocated during the block execution.  
```
__strong typeof(weakSelf) strongSelf = weakSelf;
[strongSelf doSomething];
```